tests/docker-compose.yml

```diff
@@ -2,20 +2,40 @@ version: '3.1'
 
 services:
 
-  db:
-    image: postgres:9.0
+  db90:
+    image: postgres:9.0.22
```

Comment:
Some projects provide "rolling tags" so you don't have to pin the image to a patch (e.g. `9.0` should automatically be the latest `9.0.x`. Is there a reason to not use this?

Reply:
@roobre Postgres images do not seems to have rolling tags :(

Reply:
Doesn't this one work: https://hub.docker.com/_/postgres?tab=tags&page=1&ordering=last_updated&name=9.0?
I seem to be able to `docker pull postgres:9.0`

Reply:
Tested. These are rolling tags version 💯